infra/scripts/subnet.toml

```diff
@@ -0,0 +1,57 @@
+########################################################################################################################
+# Child subnet peer
+########################################################################################################################
+
+
+[tasks.child-validator]
+env = {"BASE_DIR"="${HOME}/.ipc/${NODE_NAME}", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_DIR}/cometbft", "FM_DIR"="${BASE_DIR}/fendermint", "CMT_CONTAINER_NAME" = "cometbft-${NODE_NAME}", "ETHAPI_CONTAINER_NAME" = "eth-${NODE_NAME}", "FM_CONTAINER_NAME"= "fendermint-${NODE_NAME}", "GENESIS_FILE"="${FM_DIR}/genesis.json" }
+
+run_task = "validator-run"
+
+[tasks.child-validator-down]
+env = {"BASE_DIR"="${HOME}/.ipc/${NODE_NAME}", "NETWORK_NAME"="${NETWORK_NAME}", "CMT_DIR" = "${BASE_DIR}/cometbft", "FM_DIR"="${BASE_DIR}/fendermint", "CMT_CONTAINER_NAME" = "cometbft-${NODE_NAME}", "ETHAPI_CONTAINER_NAME" = "eth-${NODE_NAME}", "FM_CONTAINER_NAME"= "fendermint-${NODE_NAME}", "GENESIS_FILE"="${FM_DIR}/genesis.json" }
+
+run_task = "testnode-down"
+
+[tasks.validator-run]
+workspace = false
+dependencies = [
+    "testnode-down",
+    "node-init",
+    "testnode-network-create",
+    "cometbft-init",
+    "fendermint-deps",
+    "subnet-config",
+    "fendermint-start-ipc",
+    "cometbft-start",
+    "cometbft-wait",
+    "ethapi-start"
+]
+
+# Subnet validator
+# ENV variable for where things are put.
+# Merge everything and test
+[tasks.subnet-config]
+dependencies = [
+    "subnet-genesis",
+    "subnet-fm-prepare",
+]
+
+
+[tasks.subnet-genesis]
+cwd = "./target/release"
+script = """
+
+./fendermint --network=${NETWORK_TYPE} genesis --genesis-file ${GENESIS_FILE} ipc from-parent --subnet-id ${CHAIN_NAME} -p ${PARENT_ENDPOINT}  --parent-gateway ${PARENT_GATEWAY}  --parent-registry ${PARENT_REGISTRY} --base-fee ${BASE_FEE} --power-scale ${POWER_SCALE}
```

Comment:
Do these scripts run on the host, or inside a container?

Reply:
They run on the host, I followed what @dnkolegov was doing in the rest of his infra scripts, but I see that Raul already opened a PR to fix this: https://github.com/consensus-shipyard/fendermint/pull/329

I will let Raul fix that one and I will propagate the fix here.